ui/packages/consul-ui/tests/steps/interactions/form.js

```diff
@@ -10,7 +10,9 @@ export default function(scenario, find, fillIn, triggerKeyEvent, currentPage) {
       }
       return page;
     } else {
-      return page.fillIn(name, value);
+      const $el = document.querySelector(`[name="${name}"]`);
+      fillIn($el, value);
+      return page;
```

Comment:
As mentioned in the commit, for some reason ember-cli-page-object 'fillables' just stopped working, no idea why, I switched back to ember test helpers instead which I believe is more up to date than ember-cli-page-object things.